apps/root/src/common/components/token-selector/index.tsx

```diff
@@ -66,7 +66,7 @@ export const TokenSelectorItem = ({
           <Typography variant="bodySmallSemibold" color={colors[mode].typography.typo2}>
             {token.name}
           </Typography>
-          <Typography variant="bodySmallLabel" color={colors[mode].typography.typo3}>
+          <Typography variant="labelRegular">
```

Comment:
<img width="1163" alt="Screenshot 2024-09-25 at 12 46 30 am" src="https://github.com/user-attachments/assets/2104e364-045a-48f0-a987-1557276672dc">


Isn't this one bodySmall?

Reply:
Good catch sir